packages/agreement-process/src/model/domain/validators.ts

```diff
@@ -362,15 +349,13 @@ export const validateActivationOnDescriptor = (
   return descriptor;
 };
 
-const attributesSatisfied = <
-  T extends RevocableTenantAttribute | NotRevocableTenantAttribute
->(
+const attributesSatisfied = (
   descriptorAttributes: EServiceAttribute[][],
-  consumerAttributes: T[]
+  consumerAttributes: TenantAttribute[]
 ): boolean => {
-  const notRevocatedAttributeIds = consumerAttributes
```

Comment:
The last test I wrote spotted a bug here: this was not filtering revoked attributes.